students/karisPlath/THSensor.ino

```diff
@@ -0,0 +1,145 @@
+#include <avr/io.h>
+#include <util/delay.h>
+
+// button inits 
+const int UP = A0;          
+const int DWN = A1;           
+
+// init TH sensor
+#include "Adafruit_Si7021.h"
+Adafruit_Si7021 sensor = Adafruit_Si7021();
+
+// declare servo motor
+#include <Servo.h>
+Servo myservo;
+
+// define speed/direction for servo
+#define OPEN  95
+#define CLOSE 65
```

Comment:
How do these get interpreted as speeds/directions? Should probably link to the datasheet (here or in your notebook)